artemis-cli/src/main/java/org/apache/activemq/artemis/cli/commands/tools/XmlDataImporter.java

```diff
@@ -444,33 +444,59 @@ private void processMessageBody(Message message) throws XMLStreamException, IOEx
          }
       }
       reader.next();
+      ActiveMQServerLogger.LOGGER.debug("XMLStreamReader impl: " + reader);
```

Comment:
use  if (log.isDebug()) log.debug(...)

And get a logger for this class.

This is change I have made in all the longer not long ago.